source/fundamentals/builders/sort.txt

```diff
@@ -11,3 +11,346 @@ Sorts Builders
    :class: singlecol
 
 .. _sorts-builders:
+
+Overview
+--------
+
+In this guide, we show you how to use **builders** to specify
+**sort criteria** for your queries.
```

Comment:
Nitpick:
Since we have multiple builders, perhaps we could emphasize "sort criteria" first by reordering the sentence. E.g.
"In this guide, we show you how to specify sort criteria for your queries using builders." This would also reflect the order in which you explain the emphasized terms below.

Reply:
Thanks for the note. I agree the changed order better reflects the overall IA of the site and the way I presented content. Fix: https://github.com/mongodb/docs-java/pull/59/commits/b645ebd3cfc6ba417843b29106c201fb9fe6d2de

Going forward, I will try to have structure of my sentences reflect overall structure of site and organization of page.